test/libsolidity/syntaxTests/storageLayoutSpecifier/user_defined_value_type_wrap.sol

```diff
@@ -1,4 +1,4 @@
 type MyUint is uint128;
 contract C layout at MyUint.wrap(42) {}
 // ----
-// TypeError 6396: (45-60): The base slot of the storage layout must evaluate to a rational number.
+// TypeError 1763: (45-60): The base slot of the storage layout must evaluate to an integer.
```

Comment:
This is also a case where the user might not understand why we don't see the value as an integer. It could use a special message just like `address` and `bytesN`.